graph/pkg/service/v0/users.go

```diff
@@ -211,12 +218,18 @@ func (g Graph) PatchUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var (
+		changedFeature string
+		newValue       string
+	)
```

Comment:
Actually It's possible to update other attfibutes than just mail. The current implementation allows to update `displayName` as well. It's just there there is no validation of the `displayName` value happening here currently (as virtually any string is allowed for `displayName`.

This also makes me wonder how we should handle updates of multiple attributes with a single PATCH request? Would it emit multiple `UserFeatureChanged` Events? Or does the Event to have to support a list of changes?

Reply:
Can't we add all the changes to the event and have the logic of making it user readable in the audit service? I don't see the point in sending multiple events for a single PatchUser operation.

Reply:
Sure we can do that too.

Reply:
Ok, I implemented the changes.